content/docs/tools/related-tools.md

```diff
@@ -53,6 +53,8 @@ People have written tools on top of Cucumber. These are the known third-party to
 | [EmailSpec](https://github.com/bmabey/email-spec)                                                               | Collection of [RSpec](https://rspec.info) matchers and Cucumber Steps for testing email in a Rails app                                                                                                                                                                                        |
 | [fbrp](https://github.com/diabolo/fbrp/tree/master)                                                             | Sample Rails app of Cucumber showing how to use with restful-authentication.                                                                                                                                                                                                                 |
 | [Features2Cards](https://github.com/brynary/features2cards)                                                     | Create PDFs from Cucumber Features and Scenarios for printing.                                                                                                                                                                                                                               |
+| [features2](https://www.npmjs.com/package/features2)  | features2 is a simple script that creates HTML and PDF based documentation from Cucumber features. Note that the documentation is generated from the source Cucumber feature files, and NOT from the test results (there are plenty of other tools that can do that).|
+
```

Comment:
There's  a slight Markdown mishap here.

If you move the `|` character (imitate the other entries), the table does not render wrong.

Hope this helps!

Reply:
Hi, I made the adjustment. Thank you